README.md

```diff
@@ -38,30 +38,6 @@ Perform a complete build of the application via
 npm run all
 ```
 
-
-## [bpmn-js](https://github.com/bpmn-io/bpmn-js) Extension
```

Comment:
Should we keep a link to the behaviors at least?

Reply:
How about that: https://github.com/camunda/camunda-bpmn-moddle/pull/112/commits/dc5d2fb063b4ea4e8f8af425fbce883552ca2233